### PR TITLE
fixes #16819 - rebuild tftp with template kinds

### PR DESCRIPTION
**Pull request by timogoebel**

### CODE REVIEW

File: app/models/concerns/orchestration/tftp.rb [hunk 1]
```diff
@@ -36,7 +36,9 @@ def tftp6
   def rebuild_tftp
     if tftp? || tftp6?
       begin
-        setTFTP
+        host.operatingsystem.template_kinds.each do |kind|
+          setTFTP(kind)
+        end
       rescue => e
```
Comment (domcleal) [inline app/models/concerns/orchestration/tftp.rb]: Can you move this _inside_ the template_kinds loop? Have a look at Orchestration::DNS for a similar example, where the rebuild method always tries to rebuild as much as possible, even if one fails, but then returns `false` on failure.

**Review by domcleal — CHANGES_REQUESTED** [review]

**Comment by lzap** [comment]
Ah, sorry about this one. This is when tests won't help much, compiler would do it.

**Comment by timogoebel** [comment]
@lzap: No worries. ;-)

@domcleal : Pushed the latest changes that should address your comment.

**Review by domcleal — APPROVED** [review]

**Comment by domcleal** [comment]
Merged, thanks @timogoebel.

**Pull request merged**